### ISSUE TITLE
About updates in adadelta

### ISSUE THREAD

**Issue by OlafLee** [BRML/climin#38]
I think in the original paper of adadelta, there is no step_rate.

**Comment by f0k** [comment]
That's right. Feel free to add a note to the documentation. In Lasagne, we have (a bit sloppy): "In the paper, no learning rate is considered (so learning_rate=1.0). Probably best to keep it at this value."
